src/main/java/uk/gov/justice/hmpps/prison/aop/AuthorisationAspect.kt

```diff
@@ -39,6 +39,11 @@ class AuthorisationAspect(
     // no code needed - pointcut definition
   }
 
+  @Pointcut("@annotation(uk.gov.justice.hmpps.prison.security.VerifyAgencyAccess) && execution(* *(String,..)) && args(prisonId,..)")
```

Comment:
this is a bit naff still referring to an agency.  Might be better with a `VerifyPrisonAccess` since it is using the `prisonId` anyway